wordpress-mobile/WordPress-iOS
# Reader Ghost loader indicator

Part of #14399

This PR adds "ghost cells" when content is being loaded in the Reader.

### To test

Delete the app before running the tests, otherwise, the ghost cells will not appear.

1. Go to the reader
2. Check that while content is being loaded the ghost cells are displayed :)

### Screenshots

#### iPhone

<img src="https://user-images.githubusercontent.com/7040243/87812618-dd9c0400-c836-11ea-8779-fb6d1866e2a5.png" width="300" />

#### iPad (Dark mode)

<img src="https://user-images.githubusercontent.com/7040243/87812692-f60c1e80-c836-11ea-99c1-f7da290c0db8.png" width="600" />

#### iPad (Light mode)

<img src="https://user-images.githubusercontent.com/7040243/87812744-08865800-c837-11ea-80d1-a419daa9b9ab.png" width="600" />

Please, test with iPhone and iPad, light mode and dark mode.

PR submission checklist:
